src/main/java/com/example/manager/AsdUserManagerApplication.java

```diff
@@ -1,13 +1,181 @@
 package com.example.manager;
 
+import java.util.Date;
+import java.util.Scanner;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+import com.example.manager.domain.User;
+import com.example.manager.controller.UserController;
+
+@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
+@EnableMongoRepositories
+public class AsdUserManagerApplication implements CommandLineRunner {
+
+    @Autowired
+    private UserController uc;
+    Scanner scn = new Scanner(System.in);
+    int cnt = 0;
+
+    public static void main(String[] args) {
+        SpringApplication.run(AsdUserManagerApplication.class, args);
+
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        showMenu();
+    }
+
+    private void showMenu() {
+        System.out.println("------------------------------------------");
+        System.out.println("++++++ Welcome to the user manager ++++++");
+        System.out.println("------------------------------------------");
+        System.out.println("Please enter number to choose an option: \n (1)Register \n (2)Login \n (3)Exit \n");
+
+        try (Scanner scn = new Scanner(System.in)) {
+            int userInput = scn.nextInt();
+
+            switch (userInput) {
+                case 1:
+                    registerMenu();
+                    break;
+                case 2:
+                    loginMenu();
+                    break;
+                case 3:
+                    System.out.println("Closed UserManager!");
+                    System.exit(0);
+                default:
+                    System.out.println("Invalid Input! Restarting....");
+                    showMenu();
+            }
+        }
+        catch (Exception e){
+            System.out.println("\nError while getting user input!\n Shutting down User Manager...\n");
+        }
+    }
+
+    private void registerMenu() {
```

Comment:
too long function, the logic for UI and for Implementation should be seperated, with a "UserService" (or RegistrationService), where the implementation is seperated and on the GUI, you only call the userService.registration with the input as parameters

Reply:
Counts for every logic in this class